packages/react-composites/src/composites/MeetingComposite/MeetingComposite.tsx

```diff
@@ -43,6 +43,49 @@ export interface MeetingCompositeProps extends BaseCompositeProps<CallCompositeI
    * URL that can be used to copy a meeting invite to the Users clipboard.
    */
   meetingInvitationURL?: string;
+  /**
+   * Call control options to change what buttons show on the meeting composite control bar.
+   */
+  meetingCallControlOptions?: MeetingCallControlOptions;
+}
+
+/**
+ * Meeting Call controls to show or hide buttons on the calling control bar.
+ *
+ * @beta
+ */
+export interface MeetingCallControlOptions
+  extends Pick<
+    CallControlOptions,
+    /**
+     * Flag that toggles the camera button in the MeetingCallControlBar component.
```

Comment:
You shouldn't need to re-document the `Pick`ed props, the interface will inherit the documentation of the original props

Reply:
okay will take those out!